go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/howeyc/fsnotify v0.0.0-20151003194602-f0c08ee9c607
 	github.com/kr/pretty v0.1.0 // indirect
 	github.com/mattn/go-isatty v0.0.4 // indirect
+	github.com/mdlayher/netlink v0.0.0-20190313131330-258ea9dff42c
```

Comment:
I went with this package after it's use in https://github.com/prometheus/node_exporter